### ISSUE TITLE
[CLI] Dataset update GraphQL error: internal service error

### ISSUE THREAD

**Issue by franklin-feingold** [OpenNeuroOrg/openneuro#1299]
[A user](https://neurostars.org/t/openneuro-cli-problem/5107/4) is trying to update their dataset. The API key works and the CLI identifies the existing files. When trying to add new files to the dataset this error occurs: 

```
(node:6019) UnhandledPromiseRejectionWarning: Error: GraphQL error: Internal Server Error
    at new ApolloError (/usr/local/lib/node_modules/openneuro-cli/node_modules/apollo-client/bundle.umd.js:92:26)
    at Object.next (/usr/local/lib/node_modules/openneuro-cli/node_modules/apollo-client/bundle.umd.js:1296:31)
    at notifySubscription (/usr/local/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:130:18)
    at onNotify (/usr/local/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:165:3)
    at SubscriptionObserver.next (/usr/local/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:219:7)
    at /usr/local/lib/node_modules/openneuro-cli/node_modules/apollo-client/bundle.umd.js:1101:36
    at Set.forEach (<anonymous>)
    at Object.next (/usr/local/lib/node_modules/openneuro-cli/node_modules/apollo-client/bundle.umd.js:1100:21)
    at notifySubscription (/usr/local/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:130:18)
    at onNotify (/usr/local/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:165:3)
(node:6019) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6019) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
This error is hit before the new files are readout on the command line as being processed for uploading

The user has also shared the [full command line output](https://gist.github.com/alexfoias/32fbf4496a14f6267630e3adc94378f4)

**Comment by ckrountree** [comment]
Make sure that the user is using the most up-to-date node.js (v10).

**Comment by franklin-feingold** [comment]
updating node did not fix this issue

**Comment by ckrountree** [comment]
This should be addressed with #1303

**Comment by nellh** [comment]
This should be solved with the new storage backend deployment.
